code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -611,10 +619,11 @@
 	if(!z)
```

Comment:
at the proc `/proc/KillEveryoneOnZLevel(z)`, the variable name "z" should be z_category (or something like) that doesn't say just z.

Reply:
my mobile is not cooperative. it's the line above.